packages/matchbox/src/components/Inline/Inline.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Box } from '../Box';
+import styled from 'styled-components';
+import propTypes from '@styled-system/prop-types';
+import { negativeTop, negativeLeft, alignChildren } from './styles';
+
+// Negates children padding top
+const OuterWrapper = styled('div')`
+  padding-top: 1px;
+  &:before {
+    display: block;
+    content: "";
+    ${negativeTop}
+  }
+`;
+
+// Negates children padding left and aligns content
+const InnerWrapper = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+  ${negativeLeft}
+  ${alignChildren}
+`;
+
+function Inline(props) {
+  const { children, align, space = '400' } = props;
+  const items = React.Children.toArray(children);
+
+  if (items.lengh <= 1 && !align) {
```

Comment:
Typo here!

Reply:
Also, not sure what the purpose is here? Is there a scenario where the user would not pass in any items? Could `children` and `align` just be required props to handle it?

Reply:
Maybe this is a premature optimization, but I'm anticipating components like the CombobBox (tag wrapper) using this, so i don't think we could make children required

Reply:
Hmm, how do those work now? Just content rendered through a prop? I would think we may want to refactor that to use children but not 100% familiar. I trust your judgment on this.

Reply:
It does use `props.children` through `React.Children.toArray`, but children could be null. I guess the align prop is unnecessary though